resources/lang/en-US.yml

```diff
@@ -1792,4 +1792,6 @@ msg_err_cannot_join_nation_over_resident_limit: '&cYou cannot join this nation a
 msg_err_cannot_join_nation_over_town_limit: '&cYou cannot join this nation, it has reached the maximum number of allowed towns-per-nation: %s.'
 
 #Message shown confirming /tw set usedefaults
-confirmation_are_you_sure_you_want_to_reset_this_worlds_settings: 'Are you sure you want to reset all of this world''s settings to the ones which are set in the config?'
\ No newline at end of file
+confirmation_are_you_sure_you_want_to_reset_this_worlds_settings: 'Are you sure you want to reset all of this world''s settings to the ones which are set in the config?'confirmation_are_you_sure_you_want_to_reset_this_worlds_settings: 'Are you sure you want to reset all of this world''s settings to the ones which are set in the config?'
```

Comment:
Something happened to this lang string, its doubled up.

Reply:
Fixed doubled up lang string in new commit.